### ISSUE TITLE
Design our software testing procedure

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#11]
## User story
1. As a software developer 
2. I want / need a proper software testing procedure 
3. So that errors are found early and can be fixed easily 

## Acceptance criteria
* Testing procedure can handle the multiple programming languages used

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
